pkg/util/mount/mount.go

```diff
@@ -25,7 +25,7 @@ import (
 	"k8s.io/kubernetes/pkg/util/exec"
 )
 
-type Interface interface {
+type Mounter interface {
```

Comment:
re: volume.Plugin -- this would work even with the proliferation of new interfaces (PVPlugin, RecyclablePlugin, etc).   The base plugin is Plugin.